package.json

```diff
@@ -46,14 +46,17 @@
     "@mdx-js/react": "^1.6.22",
```

Comment:
You can remove `@mdx-js/mdx` and `@mdx-js/react` now, then `npm install` again, and it works!

Especially the first is causing a problem: it has every dependency locked. I’ve published new fixes and features to low-level modules (in this specific case `mdast-util-to-hast`). Because `@mdx-js/mdx` has them locked, their not coming through. Removing `@mdx-js/mdx` lifts that lock and gives you fixes and features!

Reply:
Thanks! 👍